zivid_camera/src/zivid_camera.cpp

```diff
@@ -93,440 +206,535 @@ Zivid::Application makeZividApplication()
 
 namespace zivid_camera
 {
-ZividCamera::ZividCamera(ros::NodeHandle& nh, ros::NodeHandle& priv)
-  : nh_(nh)
-  , priv_(priv)
-  , camera_status_(CameraStatus::Idle)
-  , use_latched_publisher_for_points_xyz_(false)
-  , use_latched_publisher_for_points_xyzrgba_(false)
-  , use_latched_publisher_for_color_image_(false)
-  , use_latched_publisher_for_depth_image_(false)
-  , use_latched_publisher_for_snr_image_(false)
-  , use_latched_publisher_for_normals_xyz_(false)
-  , image_transport_(nh_)
-  , zivid_(makeZividApplication())
-  , header_seq_(0)
-{
-  ROS_INFO("Zivid ROS driver version %s", ZIVID_ROS_DRIVER_VERSION);
-
-  ROS_INFO("Node's namespace is '%s'", nh_.getNamespace().c_str());
-  if (nh_.getNamespace() == "")
+
+namespace ParamNames
+{
+constexpr auto serial_number = "serial_number";
+constexpr auto file_camera_path = "file_camera_path";
+constexpr auto frame_id = "frame_id";
+}  // namespace ParamNames
+
+template <typename SettingsType>
+class CaptureSettingsController
+{
+public:
+  explicit CaptureSettingsController(rclcpp::Node & node)
+  : node_(node),
+    file_path_param_{baseName() + std::string{"_file_path"}},
+    yaml_string_param_{baseName() + std::string{"_yaml"}}
+  {
+    using namespace std::placeholders;
```

Comment:
can remove this `using`

Reply:
809f259825b4e23f1e089907fcbf3ef56eaf8cf9